ajax/libs/stacktable.js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "stacktable.js",
+  "version": "1.0.2",
+  "description": "jQuery plugin for stacking tables on small screens",
+  "filename": "stacktable.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/johnpolacek/stacktable.js.git"
+  },
+  "keywords": [
+    "js",
+    "javascript",
+    "jquery",
+    "tables"
+  ],
+  "author": {
+    "name": "John Polacek",
+    "url": "http://www.johnpolacek.com"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
```

Comment:
https://github.com/johnpolacek/stacktable.js/blob/master/License.txt#L24
Please confirm with the author if it's under `GPL-3.0` or `LGPL-3.0`.